css/style.css

```diff
@@ -6,3 +6,56 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+ header, footer {
+    margin: 0 2em;
+    padding: 1em 0em;
+    background-color: aquamarine;
+ }
+
+ h1 {
+    font-size: 40px;
+ }
+
+ img {
+    width: 40rem;
+    height: 18rem;
+    display: block;
+    margin: 2rem 0;
+ }
+
+
+ body {
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    background-color: lightcyan;
+ }
+
+ .menu {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+ }
+
+ button {
+    padding: 1rem 1rem;
+    margin: 1rem;
+    font-size: 20px;
+    border-radius: 10px;
+ }
+
+ main {
+    padding: 2rem;
+ }
+
+ article {
+    margin: 5rem 0;
+ }
+
+ 
+ article:first-child {
+    font-size: larger;
+    color: darkblue;
+ } 
+
+ footer {
+    text-align: center;
+ }
```

Comment:
The styles are good. Maybe try and do it with the css class like you did on line 32